api/python/quilt3/main.py

```diff
@@ -332,6 +334,12 @@ def create_parser():
         type=str,
         required=False,
     )
+    install_p.add_argument(
+        "--path",
+        help="If specified, only the sub-package or the entry specified by PATH will be downloaded.",
```

Comment:
```suggestion
        help="If specified, downloads only the children of PATH.",
```

Reply:
Is it still clear (was it?) that the path can be a full path to a single file?

Reply:
Rewritten.